pkg/log/status.go

```diff
@@ -202,15 +202,24 @@ func Spinner(status string) *Status {
 	return s
 }
 
-// Statusf creates a spinner, sets the prefix then returns it.
+// Spinnerf creates a spinner, sets the prefix then returns it.
 // Remember to use .End(bool) to stop the spin / when you're done.
 // For example: defer s.End(false)
+// Spin parameter is used to start the initial spin. This helpful
```

Comment:
What `Spin` parameter? This function doesn't have it.

Reply:
Argh. I forgot to remove THAT comment. Done!